src/components/Sprints/SubtaskMemberSelectionComponent.js

```diff
@@ -0,0 +1,160 @@
+import { useEffect, useReducer } from "react";
+import { Modal, Button, MenuItem, Select, makeStyles, Card, CardHeader, CardContent, Typography } from "@material-ui/core";
+import * as dbUtils from "../../utils/dbUtils";
+import "../../App.css";
+
+const useStyles = makeStyles({
+  formControl: {
+    minWidth: "50%",
+  },
+  inputLabel: {
+    color: "#bbb",
+  },
+  userInput: {
+    color: "secodary",
+    float: "right"
+  },
+
+  header:{
+   textAlign:"Center",
+   padding:"1vh",
+   fontSize:"h2"
+  },
+  closeButton:{
+    margin:"2vh"
+  },
+  modal: {
+    backgroundColor:"secondary",
+    maxWidth: '1000px',
+    minWidth: '400px',
+    width: '80%',
+    margin: 'auto',
+    marginTop: '30px',
+    marginBottom: '30px',
+    overflow: 'auto',
+  },
+});
+
+const SubTaskList = (props) => {
```

Comment:
you might want to bring this in as ({selectedStory})
By using props you would have to use props.selectedStory on every call

Reply:
also do ideally separate these components into different files